### PR TITLE
Add CustomerRequest email and description to Quotes Index view

### PR DESCRIPTION
**Pull request by ceedubs84**
https://trello.com/c/1G6J9HsU

- Added Customer Request customer email and description to Quotes Index view
- Added CSS to qoutes.scss file to make new info stand out

### CODE REVIEW

File: app/assets/stylesheets/quotes.scss [hunk 1]
```diff
@@ -1,3 +1,8 @@
 // Place all the styles related to the Quotes controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+@import "colors";
+
+.qoute_cr_header {
```
Comment (gatorjuice) [inline app/assets/stylesheets/quotes.scss]: nit, but we should stick with '-' instead of underscores for css selectors

Comment (ceedubs84) [inline app/assets/stylesheets/quotes.scss]: Cool beans. Resolved!

File: app/views/quotes/index.html.erb [hunk 2]
```diff
@@ -7,6 +7,10 @@
         <h4><%= cr.description %></h4>
           <thead>
             <tr>
+              <th class="qoute_cr_header"><%= cr.customer.email %></th>
```
Comment (gatorjuice) [inline app/views/quotes/index.html.erb]: Gotta change them here too

Comment (ceedubs84) [inline app/views/quotes/index.html.erb]: Doh!

**Comment by gatorjuice** [comment]
Thumbs up

**Pull request merged**